pom.xml

```diff
@@ -465,11 +465,6 @@
                 <artifactId>shrinkwrap-resolver-impl-maven</artifactId>
                 <version>${shrinkwrap.resolver.version}</version>
             </dependency>
-            <dependency>
-                <groupId>org.jboss.shrinkwrap.resolver</groupId>
```

Comment:
What's the reason for this removal?

Reply:
It must have come from the diff of my previous 4.0 branch work against master. The org.jboss.shrinkwrap.resolver:shrinkwrap-resolver-spi dependency is redundant here because the org.jboss.shrinkwrap.resolver:shrinkwrap-resolver-impl-maven dependency pulls it in.